### PR TITLE
Update index.markdown

### PR DESCRIPTION
**Pull request by yawhide**

### CODE REVIEW

File: index.markdown [hunk 1]
```diff
@@ -621,6 +621,11 @@ Capital One credit card that's directly targeted for people with no credit
 history/newcomers. Beware this card is free for the first year, but costs around
 $30 per year after that.
 
+#### Wells Fargo
+
+Stefan got declined twice trying to get a secured credit card there. Both times they 
+said my SSN was invalid. If you want a CC, wouldn't recommend Wells Fargo.
```
Comment (sholiday) [inline index.markdown]: Did they read the SSN off your card? Does your SSN have credit history? Do you have credit cards associated with with that SSN?

I know others who've found Wells Fargo really accommodating to Canadian interns.

File: index.markdown [hunk 2]
```diff
@@ -1123,6 +1132,18 @@ Pretty much anything on Castro Street.
 
 Everything here is pretty much centered around University Ave.
 
+#### San Mateo
+
+Restaurant heaven. Did a 8 month co-op here for a startup. 4 by 4 block of
```
Comment (sholiday) [inline index.markdown]: Can you remove "Did a 8 month co-op here for a startup. 4 by 4 block of just pure restaurants (all within 2-8 minutes walk of the caltrain station) my favorites"?

File: index.markdown [hunk 3]
```diff
@@ -1123,6 +1132,18 @@ Pretty much anything on Castro Street.
...
+#### San Mateo
+
+Restaurant heaven. Did a 8 month co-op here for a startup. 4 by 4 block of 
+just pure restaurants (all within 2-8 minutes walk of the caltrain station)
+my favorites:
+- [Bon Chon](http://us.bonchon.com/) Korean chicken wings and drumsticks. So good
+- [Mango Garden](http://www.yelp.com/biz/mango-garden-san-mateo) Go for lunch, amazing meals and portions for under 10$
+- [Ikes](http://www.ilikeikesplace.com/) Best sandwich places hands down. Huge selection, either 9$ or 10$
+- [Ben tre](http://bentrerestaurant.com/) Another amazing place (Viet food). 9-12.50$ their 5 spice chicken is the best
+- Mexican indian fusion B st between 2nd and 3rd.
```
Comment (sholiday) [inline index.markdown]: s/indian/Indian/

File: index.markdown [hunk 4]
```diff
@@ -1123,6 +1132,18 @@ Pretty much anything on Castro Street.
...
+- Lots of japanese places
```
Comment (sholiday) [inline index.markdown]: s/japanese/Japanese/

**Comment by sholiday** [comment]
Thanks for your contributions!

File: index.markdown (hunk 1 above)
Comment (yawhide) [inline index.markdown]: I entered it online, and once on the phone. My SSN has no credit history nor do I have any credit cards associated with it.

**Comment by yawhide** [comment]
@sholiday i updated this

File: index.markdown (hunk 1 above)
Comment (sholiday) [inline index.markdown]: That's to be expected right? You wont have credit history until you get a line of credit.

Generally the approach for someone new to the US is to go to their bank and get a secured credit card. Or if you've had a checking account for some time you can sometimes bring in your offer letter and get a regular card. Some times (discussed in the guide) you can even get them to look at your Canadian history.

Comment (yawhide) [inline index.markdown]: I applied for a secured credit card and got rejected twice. I should have mentioned it was a secured.

**Comment by yawhide** [comment]
sooooo

**Comment by sholiday** [comment]
Can you take out the part about the Wells Fargo credit card? Then I'll merge.
Thanks!

**Comment by yawhide** [comment]
k

**Comment by sholiday** [comment]
Thanks!

**Pull request merged**